io.aventro.CandleGenieBasic.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.aventro.CandleGenieBasic
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: io.aventro.CandleGenieBasic
+
+add-extensions:
+  org.freedesktop.Platform.Python3:
+    version: "3.11"
```

Comment:
I am not sure which doc is telling to even put this here. But stop using chatbots.